### ISSUE TITLE
Create wikipedia case chart crawler

### ISSUE THREAD

**Issue by yy** [covid19-data/covid19-data#20]
Wikipedia has country-level data, which tends to be the most up-to-date information. (WHO usually lags). If we create a crawler and keep a list of pages, then we can automatically pull Wikipedia data and override the WHO data.  

- https://en.wikipedia.org/wiki/Template:2019%E2%80%9320_coronavirus_pandemic_data/United_States_medical_cases_chart
- https://en.wikipedia.org/wiki/Template:2019%E2%80%9320_coronavirus_pandemic_data/Italy_medical_cases_chart

- [x] Write a crawler #27 
- [x] test with other countries
- [x] Expand the list of countries
- [ ] fix china table parsing.
- [ ] Try state/province-level data

**Comment by benjiqq** [comment]
wouldn't it be a lot better to have github as primary tool rather than wiki?

**Comment by yy** [comment]
@benjyz We don't have many people who can crowdsource the data here, while Wikipedia is successfully assembling the data.

**Comment by bluekura** [comment]
Does anybody who are already collecting the full list of countries that have the statistics table? If not, I will try.

**Comment by bluekura** [comment]
Now I am trying to get the entire list of the chart, yet I found weird cases. 
Some countries have the chart in the main page (2020_coronavirus_pandemic_in_{country_name}), yet the chart page does not exist (Template:2019–20_coronavirus_pandemic_data/{country_name}_medical_cases_chart)

example: 
* https://en.wikipedia.org/wiki/2020_coronavirus_pandemic_in_Singapore
* https://en.wikipedia.org/wiki/Template:2019%E2%80%9320_coronavirus_pandemic_data/Singapore_medical_cases_chart

The most strange thing is that I assumed deletion event happened, yet the history said it was not. Interesting... 

For now, I will skip those pages.

**Comment by benjiqq** [comment]
@bluekura this is the issue with wiki as a tool, its not setup for collecting data systematically

**Comment by bluekura** [comment]
@benjyz I agree. What I just meant was there is data somewhere, but not the address above. So we need to find the source (in Wikipedia, specifically) to up-to-date data from more country. I will try anyway.

**Comment by yy** [comment]
US state-level data: https://en.wikipedia.org/wiki/Template:2019%E2%80%9320_coronavirus_pandemic_data/United_States_medical_cases
